src/tests.ts

```diff
@@ -45,13 +46,15 @@ export async function testRunHandler(controller: vscode.TestController, request:
       let duration = Date.now() - starttime;
       run.passed(test, duration);
     } catch (e) {
```

Comment:
Why not cast e here?

Reply:
Same reason. Just asserts it later as `ExecResult` since that's known to be the exception type here from the `exec`.